build.sbt

```diff
@@ -82,12 +88,13 @@ lazy val root = Project("pan-domain-auth-root", file(".")).aggregate(
   panDomainAuthCore,
   panDomainAuthPlay_2_9,
   panDomainAuthPlay_3_0,
+  panDomainAuthHmac,
   panDomainAuthHmac_2_9,
   panDomainAuthHmac_3_0,
   exampleApp
 ).settings(
   publish / skip := true,
-  releaseVersion := ReleaseVersion.fromAggregatedAssessedCompatibilityWithLatestRelease().value,
+  // releaseVersion := ReleaseVersion.fromAggregatedAssessedCompatibilityWithLatestRelease().value,
```

Comment:
Still needed?

Reply:
Unfortunately we have to comment the line out until after the release due to this issue:

* https://github.com/guardian/gha-scala-library-release-workflow/issues/33

Reply:
Crikey! Thanks.